### ISSUE TITLE
Error parsing require statements

### ISSUE THREAD

**Issue by oliverbrooks** [orangemug/readme-tester#5]
If you have a readme which includes multiple require statements such as 

``` md

var thing1 = require("one-thing");
var thing2 = require("two-thing");

```

Then `readme-tester` will parse as:

``` md
var thing1 = require("/base/path/two-thing");
var thing2 = require("/base/path/");
```

Looks like the regex which parses out the `require` statements should iterate over replacing with corresponding results.

**Comment by orangemug** [comment]
@oliverbrooks I could not recreate this issue. However I did fix a very similar issue in PR #6, could you please retest and create a failing test if the issue persists

**Comment by orangemug** [comment]
I'm going to close this assuming its fixed. Feel free to re-open if it is still an issue.
